scvi/inference/experimental_inference.py

```diff
@@ -73,6 +73,22 @@ def adversarial_loss(self, tensors, *next_tensors):
     return type(self).loss(self, tensors, *next_tensors) - cls_loss
 
 
+def adversarial_loss_fish(self, tensors_seq, tensors_fish):
```

Comment:
Is it possible to put these smFISH-specific methods all the `VariationalInferenceFish` class? It'd help a lot to keep them all together.

Reply:
I'm not sure how to do this because this is specific to the adversarial inference class, I'll see if I can do something about it.